chrometool/chrometool_test.go

```diff
@@ -0,0 +1,317 @@
+package chrometool
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/chromedp/chromedp"
+	"github.com/gobwas/ws"
+	"github.com/gobwas/ws/wsutil"
+)
+
+type requestType struct {
+	method string
+	params string
+}
+
+type responseType struct {
+	index        int
+	responseList [][]byte
+}
+
+var (
+	testServerPort = flag.Int("port", 8600, "port")
+	mockChromePort = ":0"
```

Comment:
make this a flag with default value :0 the returning port logic need not change)